exchange-v2/test/contracts/v2/AssetMatcherCollectionTest.sol

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.7.6;
+
+pragma abicoder v2;
+
+import "../../../../custom-matchers/contracts/AssetMatcherCollection.sol";
```

Comment:
should be imported from @rarible

Reply:
Agree, but  I`ll do it in task https://rarible.atlassian.net/browse/RPN-968
this task is about matchers, here I prefer to work with matchers

Reply:
I mean we should do all optimisation/refactoring exchenge-v2 in another task with PR exactly